<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Elija una opcion numerada</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="num">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Grupo Numerado</string>
    </property>
    <widget class="QRadioButton" name="op3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>opcion 3</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="op1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>opcion 1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="op2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>opcion 2</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btn_aceptar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>380</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QLabel" name="mensaje_dos">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Elija una opcion de letras</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_opcionB">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>opcion B</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_opcionC">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>opcion C</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_opcionA">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>opcion A</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>367</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
